src/pages/Dashboard/Markets/styles.ts

```diff
@@ -9,7 +9,29 @@ export const useStyles = () => {
       width: 100%;
       background-color: ${theme.palette.background.paper};
       border-radius: ${theme.shape.borderRadius.large}px;
-      padding: ${theme.spacing(3, 0, 1)};
+      padding: ${theme.spacing(6)};
+    `,
+    delimiter: css`
+      margin: ${theme.spacing(6)};
+    `,
+    balance: css`
+      display: flex;
+      flex-direction: column;
+      align-items: flex-end;
+      > :first-child {
+        color: ${theme.palette.text.primary};
+      }
+    `,
+    percentOfLimit: css`
+      display: flex;
+      width: 100%;
+      align-items: center;
+      > :first-child {
+        margin-right: ${theme.spacing(2)};
+      }
+      .MuiTypography-small2 {
```

Comment:
I'm not fan of cascading like this. if we will change typography from small2 to small1 one day, this will be lost. 

I think, it would be better to use className directly or defining this directly with prop